Explorer/Assets/DCL/Browser/DecentralandUrls/DecentralandUrlsSource.cs

```diff
@@ -120,7 +120,7 @@ private static string RawUrl(DecentralandUrl decentralandUrl) =>
                 DecentralandUrl.MediaConverter => $"https://metamorph-api.decentraland.{ENV}/convert?url={{0}}",
                 DecentralandUrl.MarketplaceCredits => $"https://credits.decentraland.{ENV}",
                 DecentralandUrl.GoShoppingWithMarketplaceCredits => $"https://decentraland.{ENV}/marketplace/browse?sortBy=newest&status=on_sale&withCredits=true",
-                DecentralandUrl.EmailSubscriptions => $"https://notifications.decentraland.{ENV}",
+                DecentralandUrl.Subscriptions => $"https://notifications.decentraland.{ENV}",
```

Comment:
maybe we can rename it to notifications?

Reply:
I agree! I have unified all notifications urls in only one: https://github.com/decentraland/unity-explorer/pull/6261/commits/f8418c1a73535185c7ce73f25394bdb5190325f3